site/en/guides/border-router/docker/run.md

```diff
@@ -25,218 +19,264 @@ information.
-* Applying /etc/sysctl.d/10-network-security.conf ...
-net.ipv4.conf.default.rp_filter = 1
-net.ipv4.conf.all.rp_filter = 1
-net.ipv4.tcp_syncookies = 1
-* Applying /etc/sysctl.d/10-ptrace.conf ...
-kernel.yama.ptrace_scope = 1
-* Applying /etc/sysctl.d/10-zeropage.conf ...
-vm.mmap_min_addr = 65536
-* Applying /etc/sysctl.d/60-otbr-ip-forward.conf ...
-net.ipv6.conf.all.forwarding = 1
-net.ipv4.ip_forward = 1
-* Applying /etc/sysctl.conf ...
- * Starting userspace NAT64 tayga             [ OK ]
-/usr/sbin/service
- * Starting domain name service... bind9      [ OK ]
-/usr/sbin/service
- * dbus is not running
- * Starting system message bus dbus           [ OK ]
-   ...fail!
-otWeb[155]: border router web started on wpan0
-otbr-agent[224]: Thread interface wpan0
-otbr-agent[224]: Thread is down
-otbr-agent[224]: Check if Thread is up: OK
-otbr-agent[224]: Stop publishing service
-otbr-agent[224]: PSKc is not initialized
-otbr-agent[224]: Check if PSKc is initialized: OK
-otbr-agent[224]: Initialize OpenThread Border Router Agent: OK
-otbr-agent[224]: Border router agent started.
-```
+* `OT_RCP_DEVICE`: This specifies the connection to the Thread Radio
+  Co-Processor (RCP).
 
-OTBR Docker is now running. Leave this terminal window open and running in the
-background. If you quit the process or close the window, OTBR Docker will go
-down.
+* `OT_INFRA_IF`: This specifies the network interface used for the
+  adjacent infrastructure network (typically Wi-Fi or Ethernet).
 
-Go to [Test Connectivity](../../../guides/border-router/docker/test-connectivity.md) to
-continue with the OTBR Docker setup.
+* `OT_THREAD_IF`: This specifies the network interface name used for
+  the Thread network.
 
-## Simulated RCP
+* `OT_LOG_LEVEL`: This specifies the verbosity level of logs generated
+  by OpenThread.
 
-Use a simulated OpenThread RCP build for the simulated RCP. This is useful if
-you want to test border routing with a simulated Thread network on a single
-machine.
+## Start the OTBR Docker container
 
-### Build the simulated RCP application
+> Note: Raspberry Pi and Linux users, if you haven't done so already, make
+sure to run `sudo modprobe ip6table_filter` for OTBR firewall support.
+This allows OTBR scripts to create rules inside the Docker container
+before `otbr-agent` starts.
 
-1.  Clone the OpenThread repository:
-    ```
-    $ cd ~
-    $ git clone https://github.com/openthread/openthread
-    ```
+Create and run a new container from the OTBR image.
 
-1.  Bootstrap and build the simulated application:
-    ```
-    $ cd openthread
-    $ ./script/bootstrap
-    $ ./script/cmake-build simulation
-    ```
+```
+$ docker run --name=otbr --detach --network=host --cap-add=NET_ADMIN --device=/dev/ttyACM0 --device=/dev/net/tun --volume=/var/lib/otbr:/data --env-file=otbr-env.list --restart=always openthread/border-router
+```
 
-### Set up a bidirectional data stream
+* `docker run`: This is the base command for running a Docker
```

Comment:
I would suggest dropping the "This" at the beginning of each of these bullets.  Otherwise every sentence starts with This and it's a bit repetitive.  Maybe like:

"The base command..."
"Assigns the name..."
"Runs the container..."